packages/hydrogen/src/storefront.ts

```diff
@@ -29,18 +29,19 @@ type StorefrontApiResponse<T> = StorefrontApiResponseOk<T>;
 export type StorefrontClient = ReturnType<typeof createStorefrontClient>;
 export type Storefront = StorefrontClient['storefront'];
 
-export type CreateStorefrontClientOptions = Parameters<
+export type I18nBase = {
+  language: LanguageCode;
+  country: CountryCode;
+};
+
+export type CreateStorefrontClientOptions<TI18n extends I18nBase> = Parameters<
   typeof createStorefrontUtilities
 >[0] & {
   cache?: Cache;
   buyerIp?: string;
   requestGroupId?: string;
   waitUntil?: ExecutionContext['waitUntil'];
-  i18n?: {
-    language: LanguageCode;
-    country: CountryCode;
-    pathPrefix?: string;
-  };
+  i18n?: TI18n;
```

Comment:
@wizardlyhel I'm a bit late to the party but what's the reason to remove `pathPrefix` from here? We are still passing it in `server.ts` as a parameter to `createStorefrontClient({ i18n: {pathPrefix...}})`.

Reply:
Fix PR https://github.com/Shopify/h2/pull/432